content/blog/simple-markdown-blog-nextjs.md

```diff
@@ -159,21 +159,21 @@ export default function Meta(props) {
   return (
     <Head>
       <meta name="viewport" content="width=device-width, initial-scale=1" />
-      <meta charSet="utf-8" />
+      <meta charset="utf-8" />

```

Comment:
interesting that so many lines were changed just for this 😅

Reply:
i know ...